h2/pom.xml

```
@@ -87,6 +87,21 @@
       <artifactId>jts-core</artifactId>
       <version>1.15.0</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <version>2.9.8</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.9.8</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+      <version>2.9.8</version>
```

Q: probably it will be good to change scope to provided for this dependencies, to avoid dependency conflicts 
```suggestion
<version>2.9.8</version>
<scope>provided</scope>
```